src/lexer.cr

```diff
@@ -6,11 +6,21 @@ require "crystal/syntax_highlighter"
 module Tartrazine
   class LexerFiles
     extend BakedFileSystem
-    bake_folder "../lexers", __DIR__
+
+    macro bake_selected_lexers
+      {% for lexer in `echo -n $TT_LEXERS`.split "," %}
```

Comment:
This command won't work how you expect it to on Windows.

Reply:
Fixed following advice from the forum